src/status_im2/contexts/quo_preview/dividers/divider_label.cljs

```diff
@@ -1,54 +1,29 @@
 (ns status-im2.contexts.quo-preview.dividers.divider-label
```

Comment:
FYI: The diff in this file is just to show in this PR how the abstraction is used. Nothing of importance changed.